FIAT/brezzi_douglas_marini.py

```diff
@@ -71,16 +99,26 @@ def __init__(self, ref_el, degree):
 
 
 class BrezziDouglasMarini(finite_element.CiarletElement):
-    """The BDM element"""
+    """
+    The BDM element
+    variant can be chosen from ["point", "integral", "integral(quadrature_degree)"]
+    "point" -> dofs are evaluated by point evaluation. Note that this variant has suboptimal
+    convergence order in the H(div)-norm
+    "integral" -> dofs are evaluated by quadrature rule. The quadrature degree is high enough
+    to ensure that the interpolant of a divergence-free function is still divergence-free
```

Comment:
This is a slight lie. Just state "the quadrature degree is chosen to integrate polynomials of degree `5*k` so that most expressions will be interpolated exactly."?